### PR TITLE
Home Page for SiteName only

### PR DESCRIPTION
**Pull request by FinVamp1**
As discussed. I forked the tree rather than branch it. 
I was testing from home and got errors once I had re-directed to  the Main /sites/Sitename and so I will test again tomorrow in the office.

### CODE REVIEW

File: AppLensV2/AppLensV2.csproj [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TypeScriptToolsVersion>1.8</TypeScriptToolsVersion>
-    <ApplicationInsightsResourceId>/subscriptions/0d3ae56c-deaf-4982-b514-33d016d4a683/resourcegroups/AppLensV2/providers/microsoft.insights/components/AppLensV2</ApplicationInsightsResourceId>
+    <ApplicationInsightsResourceId>/subscriptions/4f42d231-1050-4386-b5ca-54cc38d11edf/resourcegroups/AppLensV2/providers/microsoft.insights/components/AppLensV2</ApplicationInsightsResourceId>
```
Comment (ShekharGupta1988) [inline AppLensV2/AppLensV2.csproj]: Hey Finbar, I think you need to revert this line.

File: AppLensV2/app/Home/HomeCtrl.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+module SupportCenter {
+    "use strict";
+
+    export class HomeCtrl {
+        public static $inject: string[] = ["$scope","$http", "$q", "$stateParams", "$state", "$window"];
+        constructor(private $scope, $http: ng.IHttpService, private $q: ng.IQService, private $stateParams: IStateParams, private $state: angular.ui.IStateService, private $window: angular.IWindowService) {
+        }
+
+        sendToMain() {
+            this.$stateParams.siteName = this.$scope.siteName;
```
Comment (ShekharGupta1988) [inline AppLensV2/app/Home/HomeCtrl.ts]: Dont use $scope.  If you see in app.ts, you have defined as ControllerAs for this controller as 'home'. In the html, you can simply refer to this variable as home.siteName and in controller, just use it as this.siteName.

One problem using scope is when there are hierarchical controllers, scope gets shared and its very easy to pollute it.

File: AppLensV2/app/Home/HomeCtrl.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿///<reference path="../references.ts" />
+
+module SupportCenter {
+    "use strict";
+
+    export class HomeCtrl {
+        public static $inject: string[] = ["$scope","$http", "$q", "$stateParams", "$state", "$window"];
```
Comment (ShekharGupta1988) [inline AppLensV2/app/Home/HomeCtrl.ts]: Get rid of services which you are not using. Right now, you are only using $stateParams and $state. So get rid of $scope, $http, $q and $window.

File: AppLensV2/app/Home/home.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+                <span flex></span>
+                <img src="app/assets/images/Azure-WebApps-Logo.png" class="md-avatar" alt="app" style="height:60px;width:60px" />
+            </div>
+        </md-toolbar>
+
+        <md-content layout="column" flex id="wrapper">
+            <form name="siteSearch" data-ng-submit="home.sendToMain()">
+                <md-input-container>
+                    <label>Site:</label>
+                    <input ng-model="siteName" required>
```
Comment (ShekharGupta1988) [inline AppLensV2/app/Home/home.html]: use home.siteName

**Comment by ShekharGupta1988** [comment]
Hey @FinVamp1 , can you add a screenshot of the page you added here. It will other reviewers also to understand the change.

**Comment by FinVamp1** [comment]
Thanks Shekhar, will update this all tomorrow.

**Comment by FinVamp1** [comment]
homepage Updated the items as discussed and attached a Screenshot of what the HomePage looks like.  This has changed the error when I test to this. 


TypeError: Object doesn't support property or method 'success'
   at SiteService (http://localhost:53564/app/app.js:412:13)
   at instantiate (http://localhost:53564/bower_components/angular/angular.js:4857:7)
   at Anonymous function (http://localhost:53564/bower_components/angular/angular.js:4692:7)
   at invoke (http://localhost:53564/bower_components/angular/angular.js:4842:9)
   at enforcedReturnValue (http://localhost:53564/bower_components/angular/angular.js:4676:7)
   at invoke (http://localhost:53564/bower_components/angular/angular.js:4842:9)
   at Anonymous function (http://localhost:53564/bower_components/angular/angular.js:4636:13)
   at getService (http://localhost:53564/bower_components/angular/angular.js:4783:11)
   at injectionArgs (http://localhost:53564/bower_components/angular/angular.js:4807:9)
   at instantiate (http://localhost:53564/bower_components/angular/angular.js:4854:7) <div class="ng-scope" layout="row" ui-view="" flex="" data-ng-animate="1">
angular.js (14328,11)

**Comment by ShekharGupta1988** [comment]
@FinVamp1 - Hi Finbar, changes looks good. 
The error you are seeing is most likely because we were using an older version of Angular1.5x where .success() is supported. Until we fix this, I'll share the older version of Angular1.5x that we are using and you can test this stuff.
Also, I'll go ahead and merge this PR to master but will first deploy to staging slot. If everything looks good there, we can do the swap with prod.

**Pull request merged**